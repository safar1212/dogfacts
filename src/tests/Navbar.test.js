import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../Components/Navbar';

it('Builds the snapchot of the navbar component safely', () => {
  const three = renderer
    .create(
      <Router>
        <NavBar />
      </Router>,
    )
    .toJSON();
  expect(three).toMatchSnapshot();
});
