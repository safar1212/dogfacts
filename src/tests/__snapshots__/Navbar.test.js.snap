// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Builds the snapchot of the navbar component safely 1`] = `
<nav>
  <ul>
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="arrow"
          src="arrow2.png"
        />
      </a>
    </li>
  </ul>
  <div>
    <h1>
      Dog Breeds
    </h1>
  </div>
  <div
    className="voice-setting-div"
  >
    <img
      alt="voicerecorder"
      src="mic.png"
    />
    <img
      alt="settings"
      src="settings.png"
    />
  </div>
</nav>
`;
